apps/dashboard/src/components/integrations/components/integration-general-settings.tsx

```diff
@@ -166,7 +167,13 @@ export function GeneralSettings({
               Identifier
             </FormLabel>
             <FormControl>
-              <Input id="identifier" {...field} readOnly={mode === 'update'} hasError={!!fieldState.error} />
+              <Input
+                id="identifier"
+                trailingNode={mode === 'update' ? <CopyButton valueToCopy={field.value} /> : undefined}
+                {...field}
+                disabled={mode === 'update'}
```

Comment:
I would maybe do here reado nly mode instead of disabled, so it will look better.